ci/terraform/oidc/ecc-signing-key.tf

```diff
@@ -4,10 +4,45 @@ resource "aws_kms_key" "storage_token_signing_key_ecc" {
   key_usage                = "SIGN_VERIFY"
   customer_master_key_spec = "ECC_NIST_P256"
 
+  policy = data.aws_iam_policy_document.storage_token_signing_key_access_policy.json
+
   tags = local.default_tags
 }
 
 resource "aws_kms_alias" "storage_token_signing_key_alias" {
   name          = "alias/${var.environment}-storage-token-signing-key-ecc-alias"
   target_key_id = aws_kms_key.storage_token_signing_key_ecc.key_id
-}
\ No newline at end of file
+}
+
+data "aws_iam_policy_document" "storage_token_signing_key_access_policy" {
+  statement {
+    sid    = "DefaultAccessPolicy"
+    effect = "Allow"
+
+    actions = [
+      "kms:*"
+    ]
+    resources = ["*"]
+
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"]
+    }
+  }
+
+  statement {
+    sid    = "AllowOrchAccessToKmsStorageTokenSigningKey-${var.environment}"
```

Comment:
I assume you've tested this and it's fine? I just remember that KMS SIDs are different to IAM SIDs and it's annoying